### PR TITLE
Tpwdcee 3346 paylib

### PR DESCRIPTION
**Pull request by serucee**

### CODE REVIEW

File: examples/configuration/config.php [hunk 1]
```diff
@@ -191,3 +192,11 @@
 $pbbaSecret = '	4a4396df-f78c-44b9-b8a0-b72b108ac465';
 $pbbaConfig = new PaymentMethodConfig(PayByBankAppTransaction::NAME, $pbbaMAID, $pbbaSecret);
 $config->add($pbbaConfig);
+
+
+// ### Paylib
+
+$paylibMAID = 'd79cbe98-cb2c-4882-a16e-47eaeb99ad44';
+$paylibKey = 'NO-SECRET-PROVIDED';
```
Comment (rinnhofer) [inline examples/configuration/config.php]: - clarify

File: examples/payments/Paylib/pay.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+/**
+ * Shop System SDK:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/paymentSDK-php/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/paymentSDK-php/blob/master/LICENSE
+ */
+
+// # Pay with payment method paylib
+// WARNING: This payment method is still in development, please do not use it in its current state
```
Comment (rinnhofer) [inline examples/payments/Paylib/pay.php]: I think we can remove that

File: examples/payments/Paylib/pay.php [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        $code = $status->getCode();
+        $description = $status->getDescription();
+        echo sprintf('%s with code %s and message "%s" occurred.<br>', $severity, $code, $description);
+    }
+}
+//Footer design
+require __DIR__ . '/../../inc/footer.php';
+
+
+/*
```
Comment (rinnhofer) [inline examples/payments/Paylib/pay.php]: we should remove that

File: src/Transaction/PaylibTransaction.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+namespace Wirecard\PaymentSdk\Transaction;
+
+use Wirecard\PaymentSdk\Entity\AccountHolder;
+use Wirecard\PaymentSdk\Exception\MandatoryFieldMissingException;
+
+/**
+ * Class PaylibTransaction
+ *
+ * WARNING: This payment method is still in development, please do not use it in its current state
```
Comment (rinnhofer) [inline src/Transaction/PaylibTransaction.php]: we can remove that

File: src/Transaction/PaylibTransaction.php [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    /**
+     * @return string
+     */
+    public function getEndpoint()
+    {
+        return self::ENDPOINT_PAYMENT_METHODS;
+    }
+}
```
Comment (rinnhofer) [inline src/Transaction/PaylibTransaction.php]: cs

Comment (serucee) [reply src/Transaction/PaylibTransaction.php]: Resolved with PR

**Pull request closed without merging**